rllib/connectors/tests/test_agent.py

```diff
@@ -194,14 +194,13 @@ def test_view_requirement_connector(self):
         self.assertTrue("prev_actions" in processed[0].data.for_action)
 
 
-@pytest.mark.skip(reason="activate when view_requirement is fully implemented.")
 class TestViewRequirementConnector(unittest.TestCase):
     def test_vr_connector_respects_training_or_inference_vr_flags(self):
         """Tests that the connector respects the flags within view_requirements (i.e.
         used_for_training, used_for_compute_actions) under different is_training modes.
-        For inference,
+        is_training = False
             the returned data should be state -> obs
```

Comment:
since we are here, can you help update this comment to be:
``the returned data is a SampleBatch that can be used to run corresponding policy```
and below should be:
```the returned data is the input dict itself, which the policy collector in env_runner will use to construct the episode, plus a SampleBatch that can be used to run corresponding policy```

Reply:
done.